packages/driver/test/cypress/integration/e2e/video_spec.js

```diff
@@ -0,0 +1,12 @@
+describe('video', () => {
+  it('play does not throw error on autoplay', () => {
+    cy.visit('/fixtures/video.html')
+    cy.get('video').then(($video) => {
+      // Older browsers may not return a value from play().
+      // Without --autoplay-policy=no-user-gesture-required it throws
+      // DOMException: play() failed because the user didn't interact with the document first. https://goo.gl/xX8pDD
+      return $video[0].play()
```

Comment:
I forgot to mention that a comment should have gone at the top of this spec that points to the github issue as well.

I'm merging this PR, and you can add the comment directly on `develop`